system-tests/_support/index.js

```diff
@@ -219,45 +219,56 @@ Cypress.addChildCommand("triggerHover", function(elements) {
 });
 
 /**
- * Launches a new service in group TEST_UUID using the dcos CLI
+ * Launches a new service using the dcos CLI. Service must be created
+ * in the TEST_UUID group.
  *
  * @param {Object} serviceDefinition - The service JSON definition file
  *
  */
 export function createService(serviceDefinition) {
+  const idParts = serviceDefinition.id.split("/");
+  if (idParts.length !== 3) {
+    throw new Error("Must include leading slash in service id");
```

Comment:
if `id = /$TEST_UUID/group/app` the error will be a little odd. Perhaps a better check for throwing this error is `id[0] !== '/'`.

We could then insert another check on L236:
`
if (idParts.length !== 3) {
  throw new Error("Must deploy service directly in TEST_UUID group");
}
`